api/app/Notify/Client.php

```diff
@@ -226,6 +244,113 @@ private function buildBulkPayload($name, $rows, $template, $scheduleFor = null,
         return $payload;
     }
 
+    /**
+     * Build Bulk Email Payload
+     *
+     * @param string $name Used to identify this bulk of notifications later on.
+     * @param array<string> $rows The recipient of the notification
+     * @param string $template Template of the notification
+     * @param Carbon\Carbon $scheduleFor (optional)
+     * @param string $replyTo (optional) ID for a reply to email address
+     * @return array<mixed> The partial payload
+     */
+    private function buildBulkEmailPayload($name, $rows, $template, $scheduleFor = null, $replyTo = null)
+    {
+
+        $normalizedRows = [['email address']];
+        $personalisationKeyMap = null;
+
+        if(isset($rows[0]['personalisation'])) {
+            array_push($normalizedRows[0], ...$this->buildBulkPersonalisationHeaders($rows[0]['personalisation']));
+        }
+
+        foreach($rows as $index => $row) {
+            if(!isset($row['email'])) {
+                throw new InvalidBulkRowDataException("Key 'email' not found in row $index.");
+            }
+
+            $normalizedRows[$index + 1] = array_merge(
+                [$row['email']],
+                $this->buildBulkPersonalisationRowData($row['personalisation'])
+            );
+        }
+
+        return $this->buildBulkPayload($name, $normalizedRows, $template, $scheduleFor, $replyTo);
+    }
+
+    /**
+     * Build Bulk SMS Payload
+     *
+     * @param string $name Used to identify this bulk of notifications later on.
+     * @param array<string> $rows The recipient of the notification
+     * @param string $template Template of the notification
+     * @param Carbon\Carbon $scheduleFor (optional)
+     * @param string $replyTo (optional) ID for a reply to email address
+     * @return array<mixed> The partial payload
+     */
+    private function buildBulkSmsPayload($name, $rows, $template, $scheduleFor = null, $replyTo = null)
+    {
+
+        $normalizedRows = [['phone number']];
+        $personalisationKeyMap = null;
```

Comment:
Not referenced anywhere, is this meant to be used?

Reply:
Removed in [cleanup + improve docs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5350/commits/2e8b6ca832c0d8ad2d923e837fece269b7f02663)